columnflow/production/cms/top_pt_weight.py

```diff
@@ -128,13 +142,7 @@ def top_pt_weight(self: Producer, events: ak.Array, **kwargs) -> ak.Array:
     return events
 
 
-@top_pt_weight.skip
-def top_pt_weight_skip(self: Producer) -> bool:
-    """
-    Skip if running on anything except ttbar MC simulation.
-    """
-    # never skip when there is no dataset
-    if not getattr(self, "dataset_inst", None):
-        return False
-
-    return self.dataset_inst.is_data or not self.dataset_inst.has_tag("is_ttbar")
```

Comment:
maybe make it configurable instead of plainly removing functionality? e.g. add a classattribute `ttbar_tag` that can be set to None to remove the skip func?

Reply:
Added in the last commit :+1:

Reply:
Also, I rebased on refactor/taf_init as there were conflicts to resolve.